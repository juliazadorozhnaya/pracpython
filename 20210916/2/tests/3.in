-100, 50, 53, 67, -23, 10, 5