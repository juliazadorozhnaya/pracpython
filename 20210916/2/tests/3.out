[-100, -23, 5, 10, 50, 53, 67]